README.md

```diff
@@ -256,7 +278,7 @@ These are some useful commands which help to use the script or to debug.
 
 ### Check if the script is running
 
-`svstat /service/dbus-opendtu` show if the service (our script) is running. If the number of seconds shown is low, it is probably restarting and you should look into `/data/dbus-opendtu/current.log`.
+`svstat /service/dbus-opendtu` show if the service (our script) is running. If the number of seconds shown is low, it is probably restarting and you should look into `/var/log/dbus-opendtu/current`.
```

Comment:
If you want to get the logfile with humand readable timestamp add ` | tai64nlocal` at the end.

Reply:
Currently our logs look like this (readable timestamps generated by python logging config):
```
...
@400000006654b3b90f578f14 2024-05-27 16:24:15,255 root INFO Connected to dbus, and switching over to gobject.MainLoop() (= event based)
@400000006654b3be1796b81c 2024-05-27 16:24:20,394 root WARNING Recovered inverter 0 (hoymiles2): Successfully fetched data now: Is up-to-date
@400000006654b3be18c57bec 2024-05-27 16:24:20,407 root WARNING Recovered inverter 1 (hm300): Successfully fetched data now: Is up-to-date
```

...like this using `tai64nlocal`:
```
...
2024-05-27 16:28:15.209494500 2024-05-27 16:28:15,208 root INFO Last inverter #0 '/Ac/Power': 29.8
2024-05-27 16:28:15.557006500 2024-05-27 16:28:15,546 root INFO Last inverter #1 '/Ac/Power': 153.9
```


So, @henne49, @0x7878: What do you think about removing the timestamp from our logging?

Reply:
It would save some bytes :-)

Reply:
but the tai64nlocal cannot be included in the log command? Only with post processing using tail? as I see in youd support documentation. what I would prefer readable time stamps, but with no postprocessing. 

Also can we not simply drop the timestamp from multilog?

Reply:
ok, found it it has to be in the postprocessing part